### DISCUSSION TITLE
Help with certs from windows

### DISCUSSION THREAD

**Discussion by CasperGrey** [alexandercerutti/passkit-generator#156] [Q&A] [answered]
Followed the guide and have successfully completed 

1. Created key 
2. Createrd csr with Apples details
3. Created a cert in Apples portal and downloaded cer file 
4. converted cer to pem with openSSL

I keep just getting that its an invalid PEM formatted message. I am downloading from storage into buffer as pem. do i need to convert the buffer to string as well?

**Comment by alexandercerutti** [comment]
Hi @CasperGrey, can you please share your code?

**Comment by CasperGrey** [comment, marked as answer]
Sorted by redoing the process again

**Reply by alexandercerutti** [reply]
Glad you solved! Would be awesome if you could leave a ⭐️ on the project if you like it!
